### ISSUE TITLE
Error with qgis 2.14.1 when loading plugin: int() argument must be a string or a number, not 'NoneType'

### ISSUE THREAD

**Issue by dolfandringa** [webgeodatavore/azimuth_measurement#3]
With version 0.1 and qgis 2.14.1 this plugin is reported as broken. In the plugin overview it says:
"This plugin is broken int() argument must be a string or a number, not 'NoneType'" with the plugin.

The full traceback is 
Couldn't load plugin AzimuthMeasurement due to an error when calling its classFactory() method 

```
TypeError: int() argument must be a string or a number, not 'NoneType' 
Traceback (most recent call last):
  File "/usr/share/qgis/python/qgis/utils.py", line 306, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/dolf/.qgis2/python/plugins/AzimuthMeasurement/__init__.py", line 35, in classFactory
    return AzimuthMeasurement(iface)
  File "/home/dolf/.qgis2/python/plugins/AzimuthMeasurement/azimuth_measurement.py", line 83, in __init__
    self.draw_mono_line = DrawMonoLineMapTool(self.canvas)
  File "/home/dolf/.qgis2/python/plugins/AzimuthMeasurement/draw_mono_line_map_tool.py", line 17, in __init__
    int(s.value('default_measure_color_red')),
TypeError: int() argument must be a string or a number, not 'NoneType'
```

Python version: 2.7.11 (default, Mar 31 2016, 06:18:34) [GCC 5.3.0] 
QGIS version: 2.14.1-Essen Essen, exported

**Comment by ThomasG77** [comment]
I'm unable to reproduce your issue.

Try to run from the Python console

```
from PyQt4.QtCore import SIGNAL, pyqtSignal, QSettings
s = QSettings()
s.beginGroup('Qgis')
print s.value('default_measure_color_red') # return 222
print s.value('default_measure_color_green') # return 155
print s.value('default_measure_color_blue') # return 67
s.endGroup()
```

**Comment by dolfandringa** [comment]
Hi Thomas,

I just figured out today that something went wrong with the settings in the
new QGIS version. I did s.allKeys() and very little settings were
available. But when I opened settings->options and saved it, they were all
available again in the python console. So I guess it was a problem with
QGIS, not the plugin itself, although it might make sense to provide a
default value to s.value() just in case something is the matter and the
setting isn't available.
Cheers,

Dolf.
